### ISSUE TITLE
Changing comment color

### ISSUE THREAD

**Issue by honggyukim** [namhyung/uftrace#369]
Hi,

In my working environment, the gray color isn't distinguished with normal color so I changed the gray color as follows:
```diff
$ git diff
diff --git a/utils/debug.c b/utils/debug.c
index 0218122..01c31bb 100644
--- a/utils/debug.c
+++ b/utils/debug.c
@@ -24,7 +24,7 @@
 #define TERM_COLOR_BLUE                "\033[1;34m"  /* bright blue */
 #define TERM_COLOR_MAGENTA     "\033[35m"
 #define TERM_COLOR_CYAN                "\033[36m"
-#define TERM_COLOR_GRAY                "\033[37m"
+#define TERM_COLOR_GRAY                "\033[2m"
```
The output looks like this.
![image](https://user-images.githubusercontent.com/19642328/40570590-4897d9ca-60c7-11e8-9e7c-7d4086fb28a6.png)

What do you think about this change? Is this color also working for your environment?

**Comment by honggyukim** [comment]
Here is another output when the background color is white.
![image](https://user-images.githubusercontent.com/19642328/40570660-03039b1e-60c8-11e8-8d1a-b7908e9ef3c0.png)
I think the color change works for both black and white background colors.

**Comment by honggyukim** [comment]
FYI, this is my current output. The comment color is just same as the normal color so I would like to fix it.
![image](https://user-images.githubusercontent.com/19642328/40570760-28dbe43a-60c9-11e8-9e00-0539631280ef.png)

This is shown both in my Mac terminal and putty in Windows.

**Comment by honggyukim** [comment]
I've just made a PR #370, so please take a look at it. Thanks!

**Comment by namhyung** [comment]
I'm ok with this change.

**Comment by honggyukim** [comment]
In my putty and other terminals, the comment still looks same as normal text color.  So I just made gray color to dark gray and requested in #385. So please let me know if it's okay. Thanks!

**Comment by namhyung** [comment]
Hmm.. it's hard to select a good color setup that works for everywhere..  The below link explains what those numbers mean.  It seems some terminals don't honor "faint" (2) attribute well..  Anyway I'm ok as long as it improves readability for most of users (on the default setting).

https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_(Select_Graphic_Rendition)_parameters

**Comment by namhyung** [comment]
Merged: b34a9758
